### ISSUE TITLE
Self update error: ReleaseError: No asset found for target: `x86_64-unknown-linux-gnu`

### ISSUE THREAD

**Issue by gmkey** [r-darwish/topgrade#603]
**What did you expect to happen?**
an upgrade of topgrade or a message that no newer version of topgrade exists

**What actually happened?**
it stops with this fuzzy error message:
Self update error: ReleaseError: No asset found for target: `x86_64-unknown-linux-gnu`

**Additional details:**
- Which operating system or Linux distribution are you using? 
"Ubuntu 20.04.1 LTS" 
- How did you install Topgrade?
From topgrade-v6.2.3-x86_64-unknown-linux-gnu.tar.gz binary package:
- If a specific step is failing, what happens when you run the command on your own? (Run `topgrade --dry-run` to see which commands Topgrade is running)

<details><pre>
 2021-01-14T11:42:33.216Z DEBUG topgrade::config > Configuration directory /root/.config does not exist
 2021-01-14T11:42:33.217Z DEBUG topgrade::utils  > Cannot find "notify-send"
 2021-01-14T11:42:33.217Z DEBUG topgrade         > Version: 6.2.3
 2021-01-14T11:42:33.217Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2021-01-14T11:42:33.217Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2021-01-14T11:42:33.217Z DEBUG topgrade         > Binary path: Ok("/usr/local/bin/topgrade")
 2021-01-14T11:42:33.217Z DEBUG topgrade         > Self Update: true
 2021-01-14T11:42:33.217Z DEBUG topgrade::utils  > Detected "/usr/bin/git" as "git"
 2021-01-14T11:42:33.217Z DEBUG topgrade::utils  > Cannot find "doas"
 2021-01-14T11:42:33.217Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"

―― 11:42:33 - Self update ――――――――――――――――――――――――――――――――――――――――――――――――――――――
Self update error: ReleaseError: No asset found for target: `x86_64-unknown-linux-gnu`
 2021-01-14T11:42:33.300Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2021-01-14T11:42:33.300Z DEBUG topgrade::utils  > Cannot find "powershell"
 2021-01-14T11:42:33.300Z DEBUG topgrade::runner > Step "System update"

―― 11:42:33 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-01-14T11:42:33.301Z DEBUG topgrade::utils  > Cannot find "apt-fast"
 2021-01-14T11:42:33.301Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt" "update"
Hit:1 https://repos.insights.digitalocean.com/apt/do-agent main InRelease
Get:2 http://mirrors.digitalocean.com/ubuntu focal InRelease [265 kB]
Hit:3 https://download.docker.com/linux/ubuntu focal InRelease
Hit:4 http://mirrors.digitalocean.com/ubuntu focal-updates InRelease
Hit:5 http://mirrors.digitalocean.com/ubuntu focal-backports InRelease
Get:6 http://security.ubuntu.com/ubuntu focal-security InRelease [109 kB]
Fetched 374 kB in 1s (494 kB/s)
Reading package lists... Done
Building dependency tree
Reading state information... Done
All packages are up to date.
 2021-01-14T11:42:35.745Z DEBUG topgrade::executor > Running "/usr/bin/sudo" "/usr/bin/apt" "dist-upgrade"
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
 2021-01-14T11:42:36.436Z DEBUG topgrade::runner   > Step "etc-update"
 2021-01-14T11:42:36.436Z DEBUG topgrade::runner   > Step "Brew"
 2021-01-14T11:42:36.436Z DEBUG topgrade::runner   > Step "yadm"
 2021-01-14T11:42:36.436Z DEBUG topgrade::runner   > Step "nix"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner   > Step "home-manager"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner   > Step "asdf"
 2021-01-14T11:42:36.437Z DEBUG topgrade::utils    > Path "/root/.config/emacs" doesn't exist
 2021-01-14T11:42:36.437Z DEBUG topgrade::utils    > Path "/root/.emacs.d" doesn't exist
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.doom.d does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.vim does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.config/nvim does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.zshrc does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.tmux does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.config/fish does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.config/openbox does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.config/bspwm does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.config/i3 does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::steps::git > /root/.config/sway does not exists
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "Git repositories"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "zr"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "antibody"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "antigen"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "zplug"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "zinit"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "zim"
 2021-01-14T11:42:36.437Z DEBUG topgrade::runner     > Step "oh-my-zsh"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "fisher"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "oh-my-fish"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "tmux"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "TLDR"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "pearl"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "SDKMAN!"
 2021-01-14T11:42:36.438Z DEBUG topgrade::utils      > Detected "/usr/bin/bash" as "bash"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "apm"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "rustup"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step ".NET"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "choosenim"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "cargo"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "Flutter"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "Go"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "Emacs"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "opam"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "vcpkg"
 2021-01-14T11:42:36.438Z DEBUG topgrade::runner     > Step "pipx"
 2021-01-14T11:42:36.439Z DEBUG topgrade::runner     > Step "stack"
 2021-01-14T11:42:36.439Z DEBUG topgrade::runner     > Step "tlmgr"
 2021-01-14T11:42:36.439Z DEBUG topgrade::runner     > Step "myrepos"
 2021-01-14T11:42:36.439Z DEBUG topgrade::runner     > Step "jetpack"
 2021-01-14T11:42:36.439Z DEBUG topgrade::runner     > Step "vim"
 2021-01-14T11:42:36.439Z DEBUG topgrade::utils      > Detected "/usr/bin/vim" as "vim"
 2021-01-14T11:42:36.443Z TRACE topgrade::executor   > Output of "/usr/bin/vim" "--version": Output { status: ExitStatus(ExitStatus(0)), stdout: "VIM - Vi IMproved 8.1 (2018 May 18, compiled Apr 15 2020 06:40:31)\nIncluded patches: 1-2269\nModified by team+vim@tracker.debian.org\nCompiled by team+vim@tracker.debian.org\nHuge version without GUI.  Features included (+) or not (-):\n+acl               -farsi             -mouse_sysmouse    -tag_any_white\n+arabic            +file_in_path      +mouse_urxvt       -tcl\n+autocmd           +find_in_path      +mouse_xterm       +termguicolors\n+autochdir         +float             +multi_byte        +terminal\n-autoservername    +folding           +multi_lang        +terminfo\n-balloon_eval      -footer            -mzscheme          +termresponse\n+balloon_eval_term +fork()            +netbeans_intg     +textobjects\n-browse            +gettext           +num64             +textprop\n++builtin_terms    -hangul_input      +packages          +timers\n+byte_offset       +iconv             +path_extra        +title\n+channel           +insert_expand     -perl              -toolbar\n+cindent           +job               +persistent_undo   +user_commands\n-clientserver      +jumplist          +postscript        +vartabs\n-clipboard         +keymap            +printer           +vertsplit\n+cmdline_compl     +lambda            +profile           +virtualedit\n+cmdline_hist      +langmap           -python            +visual\n+cmdline_info      +libcall           +python3           +visualextra\n+comments          +linebreak         +quickfix          +viminfo\n+conceal           +lispindent        +reltime           +vreplace\n+cryptv            +listcmds          +rightleft         +wildignore\n+cscope            +localmap          -ruby              +wildmenu\n+cursorbind        -lua               +scrollbind        +windows\n+cursorshape       +menu              +signs             +writebackup\n+dialog_con        +mksession         +smartindent       -X11\n+diff              +modify_fname      +sound             -xfontset\n+digraphs          +mouse             +spell             -xim\n-dnd               -mouseshape        +startuptime       -xpm\n-ebcdic            +mouse_dec         +statusline        -xsmp\n+emacs_tags        +mouse_gpm         -sun_workshop      -xterm_clipboard\n+eval              -mouse_jsbterm     +syntax            -xterm_save\n+ex_extra          +mouse_netterm     +tag_binary        \n+extra_search      +mouse_sgr         -tag_old_static    \n   system vimrc file: \"$VIM/vimrc\"\n     user vimrc file: \"$HOME/.vimrc\"\n 2nd user vimrc file: \"~/.vim/vimrc\"\n      user exrc file: \"$HOME/.exrc\"\n       defaults file: \"$VIMRUNTIME/defaults.vim\"\n  fall-back for $VIM: \"/usr/share/vim\"\nCompilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H   -Wdate-time  -g -O2 -fdebug-prefix-map=/build/vim-iU6mZD/vim-8.1.2269=. -fstack-protector-strong -Wformat -Werror=format-security -D_REENTRANT -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       \nLinking: gcc   -Wl,-Bsymbolic-functions -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -o vim        -lm -ltinfo -lnsl  -lselinux  -lcanberra -lacl -lattr -lgpm -ldl     -L/usr/lib/python3.8/config-3.8-x86_64-linux-gnu -lpython3.8 -lcrypt -lpthread -ldl -lutil -lm -lm      \n", stderr: "" }
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "Neovim"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "voom"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "npm"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "yarn"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "deno"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "composer"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "krew"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "gem"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "sheldon"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "rtcl"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "gcloud"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "Flatpak"
 2021-01-14T11:42:36.443Z DEBUG topgrade::runner     > Step "snap"
 2021-01-14T11:42:36.444Z DEBUG topgrade::utils      > Detected "/usr/bin/snap" as "snap"

―― 11:42:36 - snap ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-01-14T11:42:36.444Z DEBUG topgrade::executor   > Running "/usr/bin/sudo" "/usr/bin/snap" "refresh"
All snaps up to date.
 2021-01-14T11:42:36.692Z DEBUG topgrade::runner     > Step "pihole"
 2021-01-14T11:42:36.693Z DEBUG topgrade::runner     > Step "Firmware upgrades"
 2021-01-14T11:42:36.693Z DEBUG topgrade::utils      > Detected "/usr/bin/fwupdmgr" as "fwupdmgr"
 2021-01-14T11:42:36.695Z TRACE topgrade::executor   > Output of "uname" "-r": Output { status: ExitStatus(ExitStatus(0)), stdout: "5.4.0-60-generic\n", stderr: "" }
 2021-01-14T11:42:36.695Z DEBUG topgrade::steps::os::linux > Uname output: 5.4.0-60-generic

―― 11:42:36 - Firmware upgrades ――――――――――――――――――――――――――――――――――――――――――――――――
 2021-01-14T11:42:36.695Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "refresh"
Firmware metadata last refresh: 8 hours ago. Use --force to refresh again.
 2021-01-14T11:42:36.722Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "get-updates"
No updatable devices
 2021-01-14T11:42:36.749Z DEBUG topgrade::runner           > Step "Restarts"

―― 11:42:36 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
System update: OK
etc-update: SKIPPED: Cannot find "etc-update" in PATH
Brew: SKIPPED: Cannot find "/home/linuxbrew/.linuxbrew/bin/brew" in PATH
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
Git repositories: SKIPPED: No repositories to pull
zr: SKIPPED: Cannot find "zsh" in PATH
antibody: SKIPPED: Cannot find "zsh" in PATH
antigen: SKIPPED: Cannot find "zsh" in PATH
zplug: SKIPPED: Cannot find "zsh" in PATH
zinit: SKIPPED: Cannot find "zsh" in PATH
zim: SKIPPED: Cannot find "zsh" in PATH
oh-my-zsh: SKIPPED: Cannot find "zsh" in PATH
fisher: SKIPPED: Cannot find "fish" in PATH
oh-my-fish: SKIPPED: Cannot find "fish" in PATH
tmux: SKIPPED: Path "/root/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: SKIPPED: Cannot find "tldr" in PATH
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/root/.sdkman/bin/sdkman-init.sh" doesn't exist
apm: SKIPPED: Cannot find "apm" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
.NET: SKIPPED: Cannot find "dotnet" in PATH
choosenim: SKIPPED: Cannot find "choosenim" in PATH
cargo: SKIPPED: Cannot find "cargo-install-update" in PATH
Flutter: SKIPPED: Cannot find "flutter" in PATH
Go: SKIPPED: Cannot find "go" in PATH
Emacs: SKIPPED: Cannot find "emacs" in PATH
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: tlmgr must be explicity enabled in the configuration to run in Linux
myrepos: SKIPPED: Cannot find "mr" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Path "/root/.vim/vimrc" doesn't exist
Neovim: SKIPPED: Cannot find "nvim" in PATH
voom: SKIPPED: Cannot find "voom" in PATH
npm: SKIPPED: Cannot find "npm" in PATH
yarn: SKIPPED: Cannot find "yarn" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
gem: SKIPPED: Cannot find "gem" in PATH
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
Flatpak: SKIPPED: Cannot find "flatpak" in PATH
snap: OK
pihole: SKIPPED: Cannot find "pihole" in PATH
Firmware upgrades: OK
Restarts: SKIPPED: Cannot find "needrestart" in PATH
 2021-01-14T11:42:36.751Z DEBUG topgrade::terminal         > Desktop notification: Topgrade finished sucessfully
</pre></details>

**Comment by r-darwish** [comment]
Should be fixed now

**Comment by gmkey** [comment]
indeed it is!
